City-of-Helsinki/drupal-asuntotuotanto
# SATT-97: Snyk security updates + helfi packages

## Description
Updated [helfi packages](https://github.com/City-of-Helsinki/drupal-helfi-platform-config/releases), included module security updates:
- Diff
- Tfa
- Drupal core has been updated to 10.3.6

Includes both open SNYK security updates:
- #619 
- #604 

## To test

`make fresh`
- Confirm the build goes through
- Index the site and confirm the it is working as before
